ruby/lib/trailer_vote/api/issue/create.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'trailer_vote/api/issue'
+
+module TrailerVote
+  module Api
+    class Issue
+      class Create
+
+        CONTENT = MediaTypes::Issue.to_constructable.version(1).view('create')
+        SUCCESS = MediaTypes::Issue.to_constructable.version(1)
+        FAILURE = MediaTypes::Errors.to_constructable.version(1)
+
+        ACCEPT = [SUCCESS.to_s, FAILURE.to_s(0.1)].join(', ').freeze
+
+        # @private
+        def initialize(configuration:, issue:)
```

Comment:
You can't create an issue from an issue, so this should be remove. The issue is created through `#call`.

```suggestion
        def initialize(configuration:)
```